scripts/funcotator_wdl/funcotator.wdl

```diff
@@ -153,43 +157,43 @@ task Funcotate {
-     # TODO: Make this pull from google cloud, rather than from the FTP:
-     String default_datasources_version = "funcotator_dataSources.v1.6.20190124s"
+     # Mem is in units of GB but our command and memory runtime values are in MB
+     Int default_ram_mb = 1024 * 3
+     Int machine_mem = if defined(mem) then mem *1024 else default_ram_mb
+     Int command_mem = machine_mem - 1024
 
-     # You may have to change the following two parameter values depending on the task requirements
-     Int default_ram_mb = 3000
-     # WARNING: In the workflow, you should calculate the disk space as an input to this task (disk_space_gb).  Please see [TODO: Link from Jose] for examples.
-     Int default_disk_space_gb = 100
+     # Calculate disk size:
+     Float ref_size_gb = size(ref_fasta, "GiB") + size(ref_fasta_index, "GiB") + size(ref_dict, "GiB")
+     Float vcf_size_gb = size(input_vcf, "GiB") + size(input_vcf_idx, "GiB")
+     Float ds_size_gb = size(data_sources_tar_gz, "GiB")
 
-     # Mem is in units of GB but our command and memory runtime values are in MB
-     Int machine_mem = if defined(mem) then mem *1000 else default_ram_mb
-     Int command_mem = machine_mem - 1000
+     Int default_disk_space_gb = ceil( ref_size_gb + (ds_size_gb * 2) + (vcf_size_gb * 10) ) + 20
 
+     # Silly hack to allow us to use the dollar sign in the command section:
      String dollar = "$"
 
      command <<<
          set -e
          export GATK_LOCAL_JAR=${default="/root/gatk.jar" gatk_override}
 
-         # Handle our data sources:
-         DATA_SOURCES_TAR_GZ=${data_sources_tar_gz}
-         if [[ ! -e $DATA_SOURCES_TAR_GZ ]] ; then
-             # We have to download the data sources:
-             echo "Data sources gzip does not exist: $DATA_SOURCES_TAR_GZ"
-             echo "Downloading default data sources..."
-             wget ftp://gsapubftp-anonymous@ftp.broadinstitute.org/bundle/funcotator/${default_datasources_version}.tar.gz
-             tar -zxf ${default_datasources_version}.tar.gz
-             DATA_SOURCES_FOLDER=${default_datasources_version}
-         else
-             # Extract the tar.gz:
-             echo "Extracting data sources zip file..."
-             mkdir datasources_dir
-             tar zxvf ${data_sources_tar_gz} -C datasources_dir --strip-components 1
-             DATA_SOURCES_FOLDER="$PWD/datasources_dir"
+         # =======================================
+         # Hack to validate our WDL inputs:
+         #
+         # NOTE: This happens here so that we don't waste time copying down the data sources if there's an error.
+
+         if [[ "${output_format}" != "MAF" ]] && [[ "${output_format}" != "VCF" ]] ; then
+            echo "ERROR: Output format must be MAF or VCF."
          fi
 
+         # =======================================
+         # Handle our data sources:
+
+         # Extract the tar.gz:
+         echo "Extracting data sources zip file..."
```

Comment:
Not a zip file.

Reply:
Sure.  But it is a GZip file...

Either way, fixed.